### ISSUE TITLE
ORBAT: entityType and SymbolIdentifier

### ISSUE THREAD

**Issue by bergtwvd** [MSG134/NETN_FOM#6]
Although there is not a 1-1 mapping between these two, these attribute values should match up. The semantics of both attributes should point this out.

**Comment by LennartOlsson** [comment]
The EntityType specifies the type, e.g. Leopard 2, while the SymbolIdentifier specifies the class, e.g Main Battle Tank and the affiliation of a specific entity.

Yes, semantics can be extended to explain the difference.

**Comment by bergtwvd** [comment]
Is entitytype part of MSDL, or is this field added by you to enable easier instantiations of entities by a CGF?

There still needs to be an application that derives the entity type from the SIDC. So I guess we now move this mapping from each _CGF_ to the federate that manages the creation of the ORBAT object instances.

Would be good to document a default mapping somewhere. I attached a document on this topic.

[SISO-REF-010_TO_MIL-STD-2525.pdf](https://github.com/MSG134/NETN_FOM/files/3186310/SISO-REF-010_TO_MIL-STD-2525.pdf)

**Comment by LennartOlsson** [comment]
The EntityType, Deployment and Holdings was added by MSG-106 as an extension to MSDL.
 
What stands SIDC for?

**Comment by bergtwvd** [comment]
SIDC = Symbol Identification Code  (see also MIL-STD-2525).

For the additions mentioned above, these should be documented (in AMSP-04) in a ORBAT-FOM to MSDL schema traceability table. See https://github.com/MSG134/NETN_FOM/issues/13.

**Comment by LennartOlsson** [comment]
MSG-106 also added Extended description of humans and Embarkment status.

These extension are documented in the AMSP-04 in chapter 4.

**Comment by bergtwvd** [comment]
I see. I never read (and used) those parts.

A few questions pop up:
- shouldn't these additions be propagated back to SISO as change proposals to the MSDL schema?  How much are these used?
- And since we use the ORBAT FOM for exchanging ORBAT/scenario data, applications can pick there own format to save the data in. Correct?

**Comment by LennartOlsson** [comment]
Yes, if SISO is willing to do updates of the standard, it would be great.

Not all application will listen to the ORBAT FOM data published in a federation execution, they will create their entities from a file (MSDL). 
Before exercises, it is also needed to distribute the ORBAT to participants, for that is a common used format needed.

**Comment by LennartOlsson** [comment]
Semantics updated in the next version.

EntityType: DIS Enumeration, defining the type of the entity, e.g. for the Main Battle Tank type Leopard 2 A5, the value shall be: 1.1.78.1.2.5.0

SymbolIdentifier: msdl:UnitType/SymbolIdentifier, MIL 2525 C symbol code. 
Required.
The symbol code describes the class, e.g. for ARMOR TRACK, HEAVY as a Leopard 2 shall the symbol code be: S*GPUCATH-*****, the * in second (2:nd) position is to be replaced to specify the affiliation. 
The affiliation on a unit may change during an federation execution which imply that the 2:nd character will change.
Other characters specifies functionallity, e.g. if a submarine is surfaced or bottomed which can be the case during a federation execution.
